ui/DesignSystem/Primitive/Input/Text.svelte

```diff
@@ -23,19 +21,11 @@
   export let spellcheck: boolean = false;
   export let autofocus: boolean = false;
 
-  const dispatch = createEventDispatcher();
-
   // Can't use normal `autofocus` attribute on the `inputElement`:
   // "Autofocus processing was blocked because a document's URL has a fragment".
   // preventScroll is necessary for onboarding animations to work.
   $: if (autofocus) inputElement && inputElement.focus({ preventScroll: true });
 
-  const onKeydown = (event: KeyboardEvent) => {
```

Comment:
Why do you want to remove this from the input? I imagine this could be a pattern we'll use in other forms as well.